admin/class-create-block-theme-admin.php

```diff
@@ -275,15 +272,47 @@ function copy_theme_to_zip( $zip ) {
 					continue;
 				}
 
-				// Add current file to archive
 				$relative_path = substr( $file_path, strlen( $theme_path ) + 1 );
-				$zip->addFile( $file_path, $relative_path );
+				$contents = file_get_contents( $file_path );
+
+				// Replace namespace values if provided
+				if ( $new_slug ) {
+					$contents = $this->replace_namespace( $contents, $new_slug );
```

Comment:
we should only do these for certain file extensions. Right now it's trying to do it to images too. Let's go with `array( 'php', 'css', 'scss', 'js', 'txt', 'html' );` like the emptytheme script does

Reply:
Ahh, yes.  Good call.

Reply:
Done